docs/wiki/Examples.md

```diff
@@ -29,6 +29,9 @@ Use the links below to explore these examples in more detail.
 - [Override Module Role Assignments][wiki_override_module_role_assignments]
 - [Deploy Using Module Nesting][wiki_deploy_using_module_nesting]
 
+### Output
+- [Set output for reference][wiki_output_reference]
```

Comment:
I think this would be best included within the `### Intermediate (Level 200)` examples.

Please also include in the `_Sidebar.md` for navigation in the Wiki

Reply:
Thank @krowlandson for the feedback, going on leave will look at the feedback when i back from leave :-)

Reply:
Thanks @mofaizal 👍🏻